bokehjs/src/coffee/models/selections/selection.ts

```diff
@@ -0,0 +1,115 @@
+import {Model} from "../../model"
+import * as p from "core/properties"
+import {union, intersection} from "core/util/array"
+import {merge} from "core/util/object"
+import {Glyph, GlyphView} from "models/glyphs/glyph"
+
+export class Selection extends Model {
```

Comment:
You need to write type boilerplate for this and other new models (follow changes in PRs #7463 and 7467).